deploy/helm/sumologic/values.yaml

```diff
@@ -228,9 +228,7 @@ fluentd:
     totalLimitSize: "128m"
     retryMaxInterval: "10m"
     retryForever: true
-    ## usage of the gzip compression is highly not recommended due to fluentd issue:
-    ## rel: https://github.com/fluent/fluentd/issues/3056
-    compress: text
+    compress: gzip
```

Comment:
The issue from the comment doesn't seem closed/fixed: why are we changing this?

Reply:
I think I put the wrong issue in the comment 🤦 
https://github.com/fluent/fluentd/issues/3110

Reply:
Or I created new one afterward, anyway the issue is fixed and compression works properly

Reply:
I got into this issue after upgrading to latest Sumologic chart v2.1.1 with compression enabled by default. It keeps logging below warnings -

2021-04-20 21:29:59 +0000 [warn]: #0 [sumologic.endpoint.logs.default] failed to flush the buffer. retry_time=2 next_retry_seconds=2021-04-20 21:30:01 +0000 chunk="5c06a1587371d3cc21653db8ac435965" error_class=Zlib::GzipFile::Error error="not in gzip format"
  2021-04-20 21:29:59 +0000 [warn]: #0 suppressed same stacktrace
2021-04-20 21:30:01 +0000 [warn]: #0 [sumologic.endpoint.logs.kubelet] failed to flush the buffer. retry_time=3 next_retry_seconds=2021-04-20 21:30:05 +0000 chunk="5c06a1582a29b9530705397ec337ef87" error_class=Zlib::GzipFile::Error error="not in gzip format"
  2021-04-20 21:30:01 +0000 [warn]: #0 suppressed same stacktrace
2021-04-20 21:30:05 +0000 [warn]: #0 [sumologic.endpoint.logs.default] failed to flush the buffer. retry_time=4 next_retry_seconds=2021-04-20 21:30:14 +0000 chunk="5c06a1587371d3cc21653db8ac435965" error_class=Zlib::GzipFile::Error error="not in gzip format"
  2021-04-20 21:30:05 +0000 [warn]: #0 suppressed same stacktrace

Also this caused sudden dropping of messages too. So I ended up setting the buffer compression to 'text' and for plugin output, set compression to 'deflate' as per these logs,
2021-04-21 09:05:52 +0000 [info]: gem 'fluent-plugin-sumologic_output' version '1.7.1'
2021-04-21 09:05:52 +0000 [info]: gem 'fluent-plugin-systemd' version '1.0.2'
2021-04-21 09:05:52 +0000 [info]: gem 'fluentd' version '1.12.2'
/usr/local/bundle/gems/fluent-plugin-sumologic_output-1.7.1/lib/fluent/plugin/out_sumologic.rb:23:in `initialize': Invalid compression encoding text must be gzip or deflate (RuntimeError)
        from /usr/local/bundle/gems/fluent-plugin-sumologic_output-1.7.1/lib/fluent/plugin/out_sumologic.rb:209:in `new'
        from /usr/local/bundle/gems/fluent-plugin-sumologic_output-1.7.1/lib/fluent/plugin/out_sumologic.rb:209:in `configure'
        from /usr/local/bundle/gems/fluentd-1.12.2/lib/fluent/plugin.rb:178:in `configure'

Reply:
@AshutoshNirkhe Do you think that can be related to this issue?
https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1522

Reply:
Absolutely @sumo-drosiek  I think I came from that issue probably.

Reply:
Ok, we are going to track the progress there